corehq/apps/data_cleaning/views/forms.py

```diff
@@ -5,7 +5,7 @@
 from django.views.generic import TemplateView
 
 from corehq import toggles
-from corehq.apps.data_cleaning.forms import SelectCaseTypeForm
+from corehq.apps.data_cleaning.forms.setup import SelectCaseTypeForm
```

Comment:
Curious just how many forms you're expecting to create. More than ten? More than twenty? I wasn't thinking of this as a forms-heavy django app, but I don't have a sense of, like, if every filter is going to require its own form.

Reply:
quite a few. i want to separate them into columns, filters, etc. also keeping room for Form Data Cleaning